doc/connectivity/bluetooth/api/audio/bluetooth-le-audio-arch.rst

```diff
@@ -59,6 +59,292 @@ GAF has been implemented in Zephyr with the following structure.
+   | Discover All Characteristic Descriptors  | M   | M   | I   | M    |      |     | M   | M    |     |      |
+   +------------------------------------------+-----+-----+-----+------+------+-----+-----+------+-----+------+
+   | Read Characteristic Value                | M   | M   | I   | M    |      | C.3 | M   | M    |     | M    |
+   +------------------------------------------+-----+-----+-----+------+------+-----+-----+------+-----+------+
+   | Read Using Characteristic UUID           |     |     | I   |      |      | C.3 |     |      |     |      |
+   +------------------------------------------+-----+-----+-----+------+------+-----+-----+------+-----+------+
+   | Write Characteristic Value               | M   | C.3 | I   | M    |      |     | C.3 | M    |     |      |
+   +------------------------------------------+-----+-----+-----+------+------+-----+-----+------+-----+------+
+   | Write Without Response                   | M   | C.3 | I   |      |      |     | C.3 |      |     |      |
+   +------------------------------------------+-----+-----+-----+------+------+-----+-----+------+-----+------+
+   | Notifications                            | M   | M   | I   | O    |      |     | M   | M    |     |      |
+   +------------------------------------------+-----+-----+-----+------+------+-----+-----+------+-----+------+
+   | Read Characteristic Descriptors          | M   | M   | I   | M    |      |     | M   | M    |     |      |
+   +------------------------------------------+-----+-----+-----+------+------+-----+-----+------+-----+------+
+   | Write Characteristic Descriptors         | M   | M   | I   | C.4  |      |     | M   | M    |     |      |
+   +------------------------------------------+-----+-----+-----+------+------+-----+-----+------+-----+------+
+   | Read Long Characteristic Value           |     | M   | I   |      |      |     |     |      |     |      |
+   +------------------------------------------+-----+-----+-----+------+------+-----+-----+------+-----+------+
+   | Write Long Characteristic Value          | M   |     | I   |      |      |     |     |      |     |      |
+   +------------------------------------------+-----+-----+-----+------+------+-----+-----+------+-----+------+
+
+Notes:
+
+* M Mandatory
+* O Optional
+* C.1 Mandatory to support at least one Primary Service Discovery procedure
+* C.2 Mandatory to support at least one Characteristic Discovery procedure
+* C.3 Mandatory to support at least one
+* C.4 Mandatory if the Notifications sub-procedure is supported
+* C.5 Mandatory if AICS procedures are supported
+* I Inherited
+
+.. table:: GATT sub-procedures for LE Audio services
+   :widths: auto
+
+   +----------------------------------+------+------+------+-----+------+-----+-----+------+------+-----+-----+------+------+
+   | GATT sub-procedure               | AICS | ASCS | BASS | CAS | CSIS | HAS | MCS | MICS | PACS | TBS | VCS | VOCS | TMAS |
```

Comment:
Did you forget to fill out `TMAS`?

Reply:
The addition of the table is moved to PR #75570